Brocfile.js

```diff
@@ -20,4 +22,11 @@ var app = new EmberAddon();
 app.import('bower_components/bootstrap/dist/css/bootstrap.css');
 app.import('bower_components/bootstrap/dist/js/bootstrap.js');
 
-module.exports = app.toTree();
+// Copy glyphicon fonts
+var fonts = pickFiles('bower_components/bootstrap/dist/fonts', {
+  srcDir: '/',
+  files: ['**/*'],
+  destDir: '/fonts'
+});
```

Comment:
This Brocfile is for the dummy app only. Anything you want to included in the addon itself goes into `index.js`, so yes, we are good :+1: